From e49cdb52563101bfd9ddf7e8cc784b1574490204 Mon Sep 17 00:00:00 2001
From: Uri Lublin <uril@redhat.com>
Date: Sun, 14 Mar 2010 13:01:13 +0200
Subject: [PATCH 1/2] spice server: renaming library and includedir #573349

Current library's .so* files conflict with tclspice package found
in EPEL.

Renamed library's .so files (added "-server")

Include directory was "libspice", now "spice-server" for consistency.
---
 server/Makefile.am        |   14 +++++++-------
 server/spice-server.pc.in |    4 ++--
 2 files changed, 9 insertions(+), 9 deletions(-)

diff --git a/server/Makefile.am b/server/Makefile.am
index 3cd9a70..3a397c9 100644
--- a/server/Makefile.am
+++ b/server/Makefile.am
@@ -29,14 +29,14 @@ COMMON_SRCS = 				 	\
 	$(COMMON_DIR)/canvas_utils.c		\
 	$(NULL)

-lib_LTLIBRARIES = libspice.la
+lib_LTLIBRARIES = libspice-server.la

-libspice_la_LDFLAGS = 				\
+libspice_server_la_LDFLAGS =			\
 	-version-number $(SPICE_LT_VERSION)	\
 	-no-undefined				\
 	$(NULL)

-libspice_la_LIBADD = 				\
+libspice_server_la_LIBADD = 			\
 	$(GL_LIBS)				\
 	$(FFMPEG_LIBS)				\
 	$(QCAIRO_LIBS)				\
@@ -45,7 +45,7 @@ libspice_la_LIBADD = 				\
 	$(LIBRT)				\
 	$(NULL)

-libspice_la_SOURCES =				\
+libspice_server_la_SOURCES =			\
 	glz_encoder.c				\
 	glz_encoder_config.h			\
 	glz_encoder_dictionary.c		\
@@ -70,9 +70,9 @@ libspice_la_SOURCES =				\
 	vd_interface.h				\
 	$(NULL)

-nodist_libspice_la_SOURCES = $(COMMON_SRCS)
-libspiceincludedir = $(includedir)/libspice
-libspiceinclude_HEADERS =               	\
+nodist_libspice_server_la_SOURCES = $(COMMON_SRCS)
+libspice_serverincludedir = $(includedir)/spice-server
+libspice_serverinclude_HEADERS =              	\
 	spice.h					\
 	vd_interface.h				\
 	$(NULL)
diff --git a/server/spice-server.pc.in b/server/spice-server.pc.in
index 5639f4a..69c80f3 100644
--- a/server/spice-server.pc.in
+++ b/server/spice-server.pc.in
@@ -8,6 +8,6 @@ Description: SPICE server library
 Version: @VERSION@

 Requires: @SPICE_REQUIRES@
-Libs: -L${libdir} -lspice
+Libs: -L${libdir} -lspice-server
 Libs.private: @SPICE_NONPKGCONFIG_LIBS@
-Cflags: -I${includedir}/libspice
+Cflags: -I${includedir}/spice-server
-- 
1.6.6.1

